paddle/framework/net.h

```diff
@@ -101,5 +101,7 @@ class PlainNet : public Net {
   }
 };
 
+std::shared_ptr<PlainNet> AddBackwardOp(std::shared_ptr<PlainNet> ForwardOps);
```

Comment:
That's true. I remembered that `AddBackwardOp` is an interface for `PlainNet`, which generate the backward ops, then it can be used in Python. Remove it straightly is fine to me.